src/software/world/timestamped_possession_state.h

```diff
@@ -0,0 +1,106 @@
+#pragma once
+
+#include <map>
+#include <optional>
+
+#include "software/time/timestamp.h"
+#include "software/world/robot_state.h"
+#include "software/world/team_types.h"
+
+/**
+ * A light structure for a robot ID with TeamSide
+ */
+struct RobotIdWithTeamSide
```

Comment:
What do you think about putting this in `team_types.h`?

Reply:
I'm OK with that